extension/src/plots/model/collect.test.ts

```diff
@@ -367,74 +367,71 @@ describe('collectOverrideRevisionDetails', () => {
     const runningId = 'b'
     const runningGroup = `[${runningId}]`
 
-    const {
-      overrideComparison,
-      overrideRevisions,
-      unfinishedRunningExperiments
-    } = collectOverrideRevisionDetails(
-      ['a', 'b', 'c', 'd'],
-      [
-        {
-          checkpoint_tip: 'b',
-          displayColor: '#4299e1',
-          id: 'a',
-          label: 'a',
-          logicalGroupName: 'a',
-          sha: 'a',
-          status: ExperimentStatus.SUCCESS
-        },
-        {
-          checkpoint_tip: 'b',
-          displayColor: '#13adc7',
-          id: runningId,
-          label: 'b',
-          logicalGroupName: runningGroup,
-          sha: 'b',
-          status: ExperimentStatus.RUNNING
-        },
-        {
-          checkpoint_tip: 'c',
-          displayColor: '#48bb78',
-          id: 'c',
-          label: 'c',
-          logicalGroupName: 'c',
-          sha: 'c',
-          status: ExperimentStatus.SUCCESS
-        },
-        {
-          checkpoint_tip: 'd',
-          displayColor: '#f56565',
-          id: 'd',
-          label: 'd',
-          logicalGroupName: 'd',
-          sha: 'd',
-          status: ExperimentStatus.SUCCESS
-        }
-      ] as SelectedExperimentWithColor[],
-      new Set(['a', 'c', 'd', 'e']),
-      new Set(),
-      (id: string) =>
-        ({
-          [runningId]: [
-            {
-              checkpoint_tip: 'f',
-              id: 'f',
-              label: 'f',
-              logicalGroupName: runningGroup,
-              sha: 'f',
-              status: ExperimentStatus.SUCCESS
-            },
-            {
-              checkpoint_tip: 'e',
-              id: 'e',
-              label: 'e',
-              logicalGroupName: runningGroup,
-              sha: 'e',
-              status: ExperimentStatus.SUCCESS
-            }
-          ] as Experiment[]
-        }[id])
-    )
+    const { overrideComparison, overrideRevisions } =
```

Comment:
[F] Change here is that `unfinishedRunningExperiments` has been moved into `Experiments`

Reply:
otherwise the tests should be unchanged.